doc/sphinx/source/conformance.rst

```diff
@@ -113,146 +103,16 @@ Known issues related to CTS
-             powD           0.80                      {0x1.2bfb4b18164c9p+65, -0x1.b78438ae9c3bdp-8}
-            pown            0.65                      {-0x1.9p+6, -2}
-            pownD           0.62                      {-0x1.7ffffffffffffp+1, 3}
-            powr            0.82                      {0x1.91237cp-1, 0x1.4da146p+8}
-            powrD           0.80                      {0x1.2bfb4b18164c9p+65, -0x1.b78438ae9c3bdp-8}
-       remainder            0.00                      {0x0p+0, 0x0p+0}
-       remainderD           0.00                      {0x0p+0, 0x0p+0}
-          remquo        { 0.00, 0}                    0x0p+0
-          remquoD       { 0.00, 0}                    0x0p+0
-            rint            0.00                      0x0p+0
-            rintD           0.00                      0x0p+0
-           rootn            0.69                      {-0x1.e2fe6ep-74, -141}
-           rootnD           0.68                      {-0x1.8000000000001p+1, 3}
-           round            0.00                      0x0p+0
-           roundD           0.00                      0x0p+0
-           rsqrt            1.49                      0x1.019566p+124
-           rsqrtD           1.49                      0x1.01ffffffffa39p+1016
-             sin            2.48                      -0x1.09f07ap+21
-             sinD           1.87                      -0x1.f2fffffffffbap+32
-          sincos        { 2.48, 2.37}                 {0x1.09f07ap+21, 0x1.1338ccp+20}
-          sincosD       { 1.87, 2.27}                 {0x1.f2fffffffffbap+32, 0x1.d10000000074p+380}
-            sinh            2.32                      0x1.e76078p+2
-            sinhD           1.53                      -0x1.3100000000278p+4
-           sinpi            2.13                      -0x1.45f3ep-9
-           sinpiD           2.50                      -0x1.46000000000dap-7
-            sqrt            0.00                      0x0p+0
-            sqrtD           0.00                      0x0p+0
-        subtract            0.00                      {0x0p+0, 0x0p+0}
-        subtractD           0.00                      {0x0p+0, 0x0p+0}
-             tan            4.35                      -0x1.b4eba2p+22
-             tanD           4.00                      -0x1.2f000000003edp+333
-            tanh            1.18                      -0x1.ca742ap-1
-            tanhD           1.19                      0x1.f400000000395p-1
-           tanpi            4.21                      -0x1.f99d16p-3
-           tanpiD           4.09                      0x1.f6000000001d3p-3
-           trunc            0.00                      0x0p+0
-           truncD           0.00                      0x0p+0
-====================   =========================   ===========================================================
+For the sumbission, CTS was ran with the following configuration:
+OS: Ubuntu 24.04.1 LTS
```

Comment:
Does this text render as you expect, or get lumped to a single paragraph (in the HTML)?

Reply:
Good catch, I'll change it to unordered list.